lib/CodeGen/CGDebugInfo.cpp

```diff
@@ -2703,6 +2703,7 @@ llvm::DIType *CGDebugInfo::CreateTypeNode(QualType Ty, llvm::DIFile *Unit) {
   case Type::Decltype:
   case Type::UnaryTransform:
   case Type::PackExpansion:
+  case Type::TypeVariable:
```

Comment:
I don't think this is the right change for type variables.   Shouldn't it be mapped the same way as void * is mapped?